apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -425,18 +446,35 @@ function TransactionView({ txdata }: { txdata: DataType }) {
                                     styles.txsender,
                                 ])}
                             >
-                                {amount !== null && (
+                                {txdata.timestamp_ms && (
+                                    <h3
+                                        data-testid="transaction-timestamp"
+                                        className={styles.datetime}
+                                    >
+                                        {convertNumberToDate(
+                                            txdata.timestamp_ms
+                                        )}
+                                    </h3>
+                                )}
+                                {amounts !== null && (
                                     <div className={styles.amountbox}>
                                         <div>Amount</div>
                                         <div>
-                                            {presentBN(amount)}
+                                            {amounts &&
+                                                presentBN(
+                                                    amounts.reduce(
+                                                        (x, y) => x + y
+                                                    )
+                                                )}
                                             <sup>SUI</sup>
```

Comment:
Pay isn't only used for Sui, so we'll need to to handle displaying the coin symbol that was transferred.

Reply:
Thank you for letting me know about this issue; I've updated the PR to reflect this.